tck/src/main/java/org/eclipse/microprofile/opentracing/tck/OpentracingClientTests.java

```diff
@@ -63,7 +67,7 @@
     @Deployment
     public static WebArchive createDeployment() {
 
-        File[] files = Maven.resolver()
+        File[] files = Maven.resolver()qq
```

Comment:
Looks like `qq` is a typo

Reply:
thanks, we should run CI on each PR.

https://github.com/eclipse/microprofile-opentracing/pull/30